osdata/main.go

```diff
@@ -3,72 +3,71 @@ package osdata
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/shirou/gopsutil/v3/host"
+	ctypes "gohip/types"
 	"os"
 	"os/exec"
-
-	"github.com/bechampion/gohip/types"
-	ctypes "github.com/bechampion/gohip/types"
-	"github.com/shirou/gopsutil/v3/host"
 )
 
 type MainInterface struct {
-    Dst      string   `json:"dst"`
```

Comment:
Intellij did all that reformatting - not sure why every other files were seemingly formatted properly but this one ?